labs/lab-1-html-and-css/style.css

```diff
@@ -0,0 +1,114 @@
+*{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+}
+
+body {
+    background-color: #eee;
+}
+ 
+ 
+    #hireButton {
+    position:relative;
+    left:1150px;
+    top:-100px;
+    margin: 30px;
+    background-color: #000;
+    border-radius: 40px;
+    width: 140px;
+    padding: 10px 0px;
+    color: #f9fafc;
+    font-size: 16px;
+    text-align: center;
+
+}
+
+
+#scrollButton {
```

Comment:
You seem to be very stuck on `position: relative`. We talked about a bunch of different positioning tools, including `margin: auto`, `display: flex`, `display: grid`, `position: absolute` and static display with appropriate margins and padding.  `position: relative` is the least useful of the bunch.